plugins/exporter.koplugin/target/joplin.lua

```diff
@@ -15,6 +15,7 @@ local JoplinExporter = require("base"):new {
     name = "joplin",
     is_remote = true,
     notebook_name = _("KOReader Notes"),
+    version = "2.0.0",
```

Comment:
"1.0.1" or "1.1.0" as you wish. Definitly not "2.0.0"

Reply:
1.1.0 Since changes are not patch level.